content/blog/e2e-encryption/index.md

```diff
@@ -0,0 +1,65 @@
+---
+path: '/e2e-encryption'
+title: 'Adding End-to-End Encryption for Proxied Data'
+date: 2021-9-21T06:00:00.000+00:00
+featured_image: e2e-hero.png
+categories: ['Pixie Team Blogs']
+authors: ['Vihang Mehta']
+emails: ['vihang@pixielabs.ai']
+featured: true
+---
+
+End-to-end encryption has become increasingly popular as consumers demand that any data they send - a file, email, or text message - is not decipherable by any unauthorized recipients. This consumer trend is evident in the recent surge in popularity of Signal, an encrypted instant messaging service. 
+
+In this post, we’ll cover what end-to-end encryption is and walk you through how we implemented it in our system.
+
+## Why End-to-End Encryption? 
```

Comment:
I would add a preamble sentence before the dialogue. Where we say that our system is a hybrid cloud architecture, with the data collection/storage on the customer environment,  etc. It doens't have to be specific to pixie's use case, just describe the architecture and link to Michelle's blog post about hybrid cloud

Reply:
I'm struggling with this, especially because elsewhere we have tried pretty hard to not make this extremely pixie specific and left it a bit more vague.

Reply:
The way I think about "pixie-specific" is more about use case/features, and less about architecture which could apply to many different apps in different domains than us. 

We could say something like, <not exact wording> Our system is a hybrid cloud architecture where data is collected and stored on the customer environment.  The cloud component, which can be self-hosted by the user, etc